presto-mongodb/src/main/java/com/facebook/presto/mongodb/MongoSession.java

```diff
@@ -559,4 +559,18 @@ else if (value instanceof Document) {
 
         return Optional.ofNullable(typeSignature);
     }
+
+    public long deleteDocuments(SchemaTableName schemaTableName, Optional<TupleDomain<ColumnHandle>> constraint)
+    {
+        Document filter = new Document();
+        if (constraint != null) {
```

Comment:
if constrant is null, can we return early?

Reply:
We can't do like that. If the constraint is null, it deletes all the rows from the tables.